crates/db_schema/src/source/local_user.rs

```diff
@@ -10,6 +10,7 @@ use typed_builder::TypedBuilder;
 pub struct LocalUser {
   pub id: LocalUserId,
   pub person_id: PersonId,
+  #[serde(skip)]
   pub password_encrypted: String,
```

Comment:
Not sure if anything else should be skipped. This is only fetchable by that user anyway.

Reply:
Looks good.